tensorflow/examples/tutorials/word2vec/word2vec_basic.py

```diff
@@ -93,6 +93,14 @@ def build_dataset(words, n_words):
 
 
 # Step 3: Function to generate a training batch for the skip-gram model.
```

Comment:
Please don't move comments attached to one function to another function that you've just created.